.unreleased/bugfix_SDC_995

```diff
@@ -0,0 +1 @@
+Fixes: #995 Improvements in hypertable_size function 
```

Comment:
You should only mention public issues (or PRs) in changelog

Reply:
Thanks for the info, I have removed it.